honestcash-v2/src/app/wallet/wallet.module.ts

```diff
@@ -0,0 +1,38 @@
+import {NgModule} from '@angular/core';
+import {AuthWelcomeComponent} from '../auth/pages/welcome/welcome.component';
+import {WalletContainerComponent} from './wallet-container.component';
+import {CommonModule} from '@angular/common';
+import {RouterModule, Routes} from '@angular/router';
+import {FormsModule} from '@angular/forms';
+import {UnauthorizedGuard} from 'core/shared/guards/unauthorized.guard';
+import {SharedModule} from '../../core/shared.module';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: WalletContainerComponent,
+/*    children: [
```

Comment:
wallet module will have one single nicely designed wallet page (before I get Jenny's designs).
settings module will also be the same (with one page and scrolling with snapping links)
profile module will probably have the tabs structure on the contrary.